src/WP_CLI/CommandWithMeta.php

```diff
@@ -254,6 +254,127 @@ public function update( $args, $assoc_args ) {
 	}
 
 	/**
+	 * Get meta field value.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <id>
+	 * : The ID of the object.
+	 *
+	 * <key>
+	 * : The name of the meta field to get.
+	 *
+	 * <pluck-key>
+	 * : The name of the inner key to get.
+	 *
+	 * [--delimiter=<character>]
+	 * : The character to delimit hierarchy within the pluck-key.
+	 * ---
+	 * default: .
+	 * ---
+	 *
+	 * [--format=<format>]
+	 * : Accepted values: table, json. Default: table
```

Comment:
This is mostly copied from the existing `get` subcommand. The table output is a misnomer for sure as it does not output as a table.

It runs through `\WP_CLI::print_value` so it should probably be updated with the options it supports.